packages/asset-buyer/docs/reference.mdx

```diff
@@ -0,0 +1,1352 @@
+
+___
+
+###  InvalidOrderProviderResponse
+
+• **InvalidOrderProviderResponse**: = "INVALID_ORDER_PROVIDER_RESPONSE"
+
+*Defined in [types.ts:121](https://github.com/0xProject/0x-monorepo/blob/6dd77d5c8/packages/asset-buyer/src/types.ts#L121)*
+
+___
+
+###  NoAddressAvailable
+
+• **NoAddressAvailable**: = "NO_ADDRESS_AVAILABLE"
+
+*Defined in [types.ts:120](https://github.com/0xProject/0x-monorepo/blob/6dd77d5c8/packages/asset-buyer/src/types.ts#L120)*
+
+___
+
+###  NoEtherTokenContractFound
+
+• **NoEtherTokenContractFound**: = "NO_ETHER_TOKEN_CONTRACT_FOUND"
+
+*Defined in [types.ts:115](https://github.com/0xProject/0x-monorepo/blob/6dd77d5c8/packages/asset-buyer/src/types.ts#L115)*
+
+___
+
+###  NoZrxTokenContractFound
+
+• **NoZrxTokenContractFound**: = "NO_ZRX_TOKEN_CONTRACT_FOUND"
+
+*Defined in [types.ts:116](https://github.com/0xProject/0x-monorepo/blob/6dd77d5c8/packages/asset-buyer/src/types.ts#L116)*
+
+___
+
+###  SignatureRequestDenied
+
+• **SignatureRequestDenied**: = "SIGNATURE_REQUEST_DENIED"
+
+*Defined in [types.ts:123](https://github.com/0xProject/0x-monorepo/blob/6dd77d5c8/packages/asset-buyer/src/types.ts#L123)*
+
+___
+
+###  StandardRelayerApiError
+
+• **StandardRelayerApiError**: = "STANDARD_RELAYER_API_ERROR"
+
+*Defined in [types.ts:117](https://github.com/0xProject/0x-monorepo/blob/6dd77d5c8/packages/asset-buyer/src/types.ts#L117)*
+
+___
+
+###  TransactionValueTooLow
+
+• **TransactionValueTooLow**: = "TRANSACTION_VALUE_TOO_LOW"
+
+*Defined in [types.ts:124](https://github.com/0xProject/0x-monorepo/blob/6dd77d5c8/packages/asset-buyer/src/types.ts#L124)*
+
+<hr />
+
+> # Interface: AssetBuyerOpts
+
+networkId: The ethereum network id. Defaults to 1 (mainnet).
+orderRefreshIntervalMs: The interval in ms that getBuyQuoteAsync should trigger an refresh of orders and order states. Defaults to 10000ms (10s).
+expiryBufferSeconds: The number of seconds to add when calculating whether an order is expired or not. Defaults to 300s (5m).
+
+## Hierarchy
+
+* **AssetBuyerOpts**
+
+## Index
+
+### Properties
+
+* [expiryBufferSeconds](#expirybufferseconds)
+* [networkId](#networkid)
+* [orderRefreshIntervalMs](#orderrefreshintervalms)
+
+## Properties
+
+###  expiryBufferSeconds
+
+• **expiryBufferSeconds**: *number*
+
+*Defined in [types.ts:108](https://github.com/0xProject/0x-monorepo/blob/6dd77d5c8/packages/asset-buyer/src/types.ts#L108)*
+
+___
+
+###  networkId
+
+• **networkId**: *number*
+
+*Defined in [types.ts:106](https://github.com/0xProject/0x-monorepo/blob/6dd77d5c8/packages/asset-buyer/src/types.ts#L106)*
+
+___
+
+###  orderRefreshIntervalMs
+
+• **orderRefreshIntervalMs**: *number*
+
+*Defined in [types.ts:107](https://github.com/0xProject/0x-monorepo/blob/6dd77d5c8/packages/asset-buyer/src/types.ts#L107)*
+
+<hr />
+
+> # Interface: BuyQuote
+
+assetData: String that represents a specific asset (for more info: https://github.com/0xProject/0x-protocol-specification/blob/master/v2/v2-specification.md).
```

Comment:
Going forward how do we get these definitions down to the appropriate properties, do we just describe it above the property itself and it will render nicely?

For now we should respect the new line with a double space. Otherwise they all render as one big line.

Reply:
We need to investigate if there is a way to make them render properly. This is the first pass, and we'll keep iterating on it.